.github/workflows/prerelease.yml

```diff
@@ -1,19 +1,15 @@
-name: Publish @next pre-release
+name: Prerelease
+
+on: [push, pull_request]
```

Comment:
Won't this create a prerelease for every branch that is pushed to the repo?
Don't we want to limit it to branches that have PRs and a small defined set of special branches (e.g. `main`. `d1` and `workflows`)?

Reply:
Given that every PR we make is a branch it just duplicates what we are already doing for PRs